__tests__/comments.test.js

```diff
@@ -0,0 +1,29 @@
+const { getComment, getComments, getAgent } = require('../db/data-helpers');
+
+const request = require('supertest');
+const app = require('../lib/app');
+const mongoose = require('mongoose');
+
+describe('comment routes', () => {
+
+  it('creates a comment', () => {
```

Comment:
I think you're getting the jwt error because you're not sending a user to get a token. you need to create a user and send that user info through. So where you have commentBy you'll want to actually pass the user._id.